tm2/pkg/db/internal/test_common.go

```diff
@@ -0,0 +1,132 @@
+package internal
+
+import (
+	"bytes"
+	"encoding/binary"
+	"math/rand"
+	"testing"
+
+	"github.com/gnolang/gno/tm2/pkg/db"
+)
+
+const (
+	strChars = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz" // 62 characters
+)
+
+// RandStr constructs a random alphanumeric string of given length.
+func RandStr(length int) string {
```

Comment:
The existence of this entire method is most likely not required, but removing it out is out of the scope of this PR

Reply:
I mean, it's OK and it's in an `internal` package (so it cannot actually be called by any non-db package), so I think it should be fine :man_shrugging: